src/lib.rs

```diff
@@ -297,7 +297,8 @@
     allow(
         clippy::unreadable_literal,
         clippy::cognitive_complexity,
-        clippy::float_cmp
+        clippy::float_cmp,
+        clippy::match_like_matches_macro
```

Comment:
this macro forces you to write something like
```rust
match a {
    SomeEnum::B => false,
    _ => true
}
```
as 
```rust
!matches!(a, SomeEnum::B)
```
which I personally find less readable (super easy to miss that first `!`). What do you all think?